Palindrome.java

```diff
@@ -25,6 +27,64 @@
  */
 public class Palindrome {
     public static void main(String[] args) {
-        // Your code goes here :)
+
+        Scanner input  = new Scanner(System.in);
+        byte palindromeSize;
+
+        System.out.printf("Please enter a number(int), between 0 and 1,000,000,000: ");
+        long inputNum = input.nextInt();
+
+        while (inputNum <= 0 || inputNum > 1000000000) {
+            System.out.printf("Please enter a valid number: ");
+            inputNum = input.nextLong();
+        }
+        input.close();
+
+        palindromeSize = checkIfPalindrome(inputNum);

```

Comment:
We haven't learned to use methods... but a great choice

Reply:
Yes, I agree. A method should do one thing and one thing only.
I'll revise.